src/components/menus/PlanMenu.svelte

```diff
@@ -76,6 +84,13 @@
           <div class="column-name">Open parent plan</div>
         </MenuItem>
       {/if}
+      <hr class="menu-divider" />
+      <MenuItem on:click={createPlanSnapshot}>
+        <div class="column-name">Take Snapshot</div>
```

Comment:
Create or Take? Seeing Create in the modal, perhaps they should be unified?

Reply:
Ah yeah. I was just going by the figma wording. @lklyne do you have a preference?